Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -194,8 +189,7 @@ public async Task<ClientWebSocket> GetWebsocketConnection(string path = "", int
             {
                 port = tcpPort;
             }
-             
-            FindCloudletReply findCloudletReply = await me.RegisterAndFindCloudlet(orgName, appName, appVers, location, carrierName);
+            FindCloudletReply findCloudletReply = await me.RegisterAndFindCloudlet(orgName, appName, appVers, location, carrierName, developerAuthToken.Length > 1 ? developerAuthToken : null);
```

Comment:
Why greater than 1? Shouldn't this default to empty string? Same for line 354

Reply:
<img width="614" alt="Screen Shot 2020-05-15 at 2 38 03 PM" src="https://user-images.githubusercontent.com/24863504/82098408-c23b3e00-96b9-11ea-8f5d-788baba2865e.png">

Reply:
Fixed Greater than 0, it defaults to null as per the C# , Tested it and its working